src/Form/AccountForm.php

```diff
@@ -233,19 +250,72 @@ private function loadMpxUsers(FormStateInterface $form_state): array {
    *     - The account options.
    *     - The account public IDs.
    */
-  private function accountOptions(FormStateInterface $form_state): array {
-    $user_entity_id = $form_state->getValue('user');
+  protected function accountOptions(FormStateInterface $form_state): array {
+    try {
+      try {
```

Comment:
Interesting: I have never seen a try within a try.

Reply:
Yeah, I try to avoid it, but it's the only way to handle catching an exception and then deciding you need to re-throw it for a more general handler.